src/pivot-table/data/extract-headers.ts

```diff
@@ -113,11 +104,13 @@ const extractHeaders = (
     );
   }
 
-  // Update canBeResized on bottom row
-  if (matrix.length > 0) {
-    for (const cell of matrix[matrix.length - 1]) {
+  // Update canBeResized, isActivelySorted and colIdx on bottom row
+  if (matrix.length) {
+    for (const [colIdx, cell] of matrix[matrix.length - 1].entries()) {
       if (cell) {
-        cell.canBeResized = true;
+        cell.canBeResized = cell.isLeftDimension;
+        cell.isActivelySorted = colIdx === (layoutService.layout.qHyperCube.activelySortedColumn?.colIdx ?? 0);
+        cell.colIdx = cell.colIdx || colIdx;
```

Comment:
Will be reassigned if `cell.colIdx === 0`

```suggestion
        cell.colIdx = cell.colIdx ?? colIdx;
```

Reply:
not really actually. Since this is set by default to 0, it should actually change when it is 0. It is only fore pseudo dim it should not be updated